src/site/stages/build/options.js

```diff
@@ -147,26 +150,69 @@ function deriveHostUrl(options) {
   ];
 }
 
-function setUpFeatureFlags(options) {
+// Sets up the CMS feature flags by either querying the CMS for them
+// or using ../../utilities/featureFlags
+async function setUpFeatureFlags(options) {
   global.buildtype = options.buildtype;
-  const {
-    enabledFeatureFlags,
-    featureFlags,
-  } = require('../../utilities/featureFlags');
+  let enabled;
```

Comment:
is this a boolean (isEnabled) or a list of booleans (enabledDrupalFeatures)?

Reply:
A list of all the features. I'll rename. :+1: